app/views/shared/_landing_page_filter.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="organisation-container">
+  <ul class="govuk-list" data-cy="organisation-filter-list">
+    <% @organisations.each do |organisation| %>
+      <div class="govuk-grid-column-one-third filter-card">
+        <li class="organisation-item">
+          <a class="govuk-link <%=organisation.name%>-filter-link" href="<%= data_services_path(filters: organisation.id) %>" data-cy="<%= organisation.name%>-filter-link">
+            <% if organisation.name == 'Department for Work and Pensions' %>
```

Comment:
I'd avoid constants and hardcoded values. One way to add and use presenters, we can pair another time. For now I suggest you extract this in a helper.

Reply:
I have now extracted this into a helper :)